app/views/reservations/index.html.erb

```diff
@@ -1,17 +1,47 @@
+</tbody>
+</table>
 <div class="wrapper">
 <%= render partial: "shared/sidebar" %>
     <div class="main-panel">
-        <div class="content">
-            <div class="container-fluid">
-                <div class="row">
-                  <div class="col-md-7">
+      <div class="content">
+        <div class="container-fluid">
+            <div class="row">
+              <div class="col-md-7">
+                <h1>Reservations</h1>
+                <div class="card">
+                  <div class="content ">
                     <h1>Reservations</h1>
-                    <div class="card">
-                      <div class="content ">
-                        <p>Reservation stuff</p>
-                      </div>
+                    <hr>
+
+                    <table>
+                      <tr>
+                        <th>Start Date</th>
+                        <th>End Date</th>
+                        <th>First Name</th>
+                        <th>Last Name</th>
+                        <th>Email</th>
+                        <th>Phone Number</th>
+                        <th>Status</th>
+                      </tr>
+                      <tbody>
+                    <div class="all_listings">
+                        <% @listings.each do |listing| %>
+                          <% listing.reservations.each do |reservation| %>
+                            <tr>
+                            <td><%= reservation.start_date %></td>
+                            <td><%= reservation.end_date %></td>
+                            <td><%= reservation.user.first_name %></td>
+                            <td><%= reservation.user.last_name %></td>
+                            <td><%= reservation.user.email %></td>
+                            <td><%= reservation.user.phone_number %></td>
+                            <td><%= reservation.status %></td>
+                            </tr>
+                          <% end %>
+                      <% end %>
```

Comment:
This might be too much logic in the view. We could probably add that a user has many reservations through listings in the user model and then just call `reservation.each do...` here. Also, do we want the listing's title in this table so that the host can see what listing the reservation is for? @valerietrudell